<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Activity</class>
 <widget class="QDialog" name="Activity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="l_dialogTitle">
     <property name="text">
      <string>Leistung hinzuf√ºgen</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Unternehmen</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_company">
       <property name="placeholderText">
        <string>Unternehmen...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Ansprechpartner</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cb_person">
       <property name="placeholderText">
        <string>Ansprechpartner...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Wann</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_when">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Wert</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_value"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Was</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="le_what"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Infos</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QTextEdit" name="te_info"/>
     </item>
     <item row="4" column="1">
      <widget class="QCalendarWidget" name="cw_calender">
       <property name="gridVisible">
        <bool>false</bool>
       </property>
       <property name="navigationBarVisible">
        <bool>true</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_okay">
         <property name="text">
          <string>Okay</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Leistungen</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_activity">
       <property name="placeholderText">
        <string>Leistungen...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_activity</tabstop>
  <tabstop>cb_company</tabstop>
  <tabstop>cb_person</tabstop>
  <tabstop>le_when</tabstop>
  <tabstop>cw_calender</tabstop>
  <tabstop>le_value</tabstop>
  <tabstop>le_what</tabstop>
  <tabstop>te_info</tabstop>
  <tabstop>pb_okay</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
